### ISSUE TITLE
Skip Match Columns step 

### ISSUE THREAD

**Issue by rishabhjindalcred** [UgnisSoftware/react-spreadsheet-import#240]
@masiulis Could you please provide an example for the above? I am having difficulty understanding how to use the 'skip headers' feature.

**Comment by jppty** [comment]
Agreed - #226 is marked closed/completed, however I don't think it was actually addressed by #188.

In the case that you want to start with uploadStep and then skip to the validationStep, there doesn't seem to be a way.
